src/pages/javascript/comparison-and-logical-operators/index.md

```diff
@@ -1,15 +0,0 @@
----
-title: Comparison and Logical Operators
```

Comment:
The comparison part of this is indeed a duplicate, but I can't find an article on logical operators (`&&`, `||`). Instead of removing this, can you rename it to be only about logical operators?